### PR TITLE
Add hiring process to README

### PR DESCRIPTION
**Pull request by asyaisakova**
@alekseypolukeev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 ## Overview
-Challenge program gives interested individuals a chance to prove themselves and learn a bit about
-the company & products. Our challenges are extremely independent and will require you to manage 
-your own time and work process.
+[Challenge program](https://github.com/1712n/challenge) gives interested individuals a chance to prove themselves and learn a bit about
+the company & products. Our challenges are extremely independent and will require you to manage your own time and work process.
+
+### Process
+1. Submit your challenge as described in [Workflow](README.md#workflow).
+1. In case your submission is approved, one of our team members will reach out to set up 30-minute Google Meet call.
```
Comment (alekseypolukeev) [inline README.md:9]: We told about HR/Bar Raiser 15-30m and the following 1-1.5h technical interviews

File: README.md [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
+1. Candidates that made it to the final stage will be offered a 3-month full- or part-time contract with specific milestones (goals) to achieve.
```
Comment (alekseypolukeev) [inline README.md:10]: It looks like we offer only 3-month contract. Would be nice to describe it better or remove at all

File: README.md [hunk 3]
```diff
@@ -1,9 +1,13 @@
 # Binance order book metrics challenge
 
 ## Overview
-Challenge program gives interested individuals a chance to prove themselves and learn a bit about
-the company & products. Our challenges are extremely independent and will require you to manage 
-your own time and work process.
+[Challenge program](https://github.com/1712n/challenge) gives interested individuals a chance to prove themselves and learn a bit about
+the company & products. Our challenges are extremely independent and will require you to manage your own time and work process.
+
+### Process
```
Comment (alekseypolukeev) [inline README.md:7]: Let's also make "Required skills" a level 3 subheading?

**Review by asyaisakova — COMMENTED** [review]
fixed the comments

**Review by asyaisakova — COMMENTED** [review]
Added contract details

**Review by alekseypolukeev — APPROVED** [review]

**Pull request merged**